README.md

```diff
@@ -125,6 +125,35 @@ Add the `size` script to your test suite:
 If you don't have a continuous integration service running, don’t forget
 to add one — start with [Travis CI](https://github.com/dwyl/learn-travis).
 
+## Config
+Add `size-limit` section to `package.json` and `size` script:
```

Comment:
No newline after the title.

It is better to explicit say about 2 types of config.

Reply:
Is it good enough?

![image](https://user-images.githubusercontent.com/9018702/32444427-aaf135b4-c313-11e7-8bd5-58a839658a62.png)

Reply:
Yeap, looks good.